### DISCUSSION TITLE
Solis Hybrid latest firmware (Dec 2024)

### DISCUSSION THREAD

**Discussion by Ander3488** [wills106/homeassistant-solax-modbus#1159] [General]
Hi, I have a Solis Hybrid S5-EH1P6K-L and just had the latest firmware installed last night ( Inverter Version
4C004F-650000 ), which now has six charge slot times with SOC and Charge current in each slot along with six discharge slots also.
Can we have this integration updated to resolve the ability to use the new time slots please?

**Comment by wawyed** [comment]
Hey, this should be now supported if you switch the Inverter type to: solis_fb00

![image](https://github.com/user-attachments/assets/002bb780-fbad-4197-8bfb-29aeb4abfd1c)

**Comment by Ander3488** [comment]
Hi,

Thanks for the update and had already implemented this change and it all works fantastic.

Thanks again

Bob

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by wawyed** [reply]
Awesome, I just updated the firmware myself so had to find out as well :D. You wouldn't happen to have the yaml code for the dashboard with the new sensors and switches. Would be much appreciated so I don't have to manually do it
